client/src/components/Events/Events.jsx

```diff
@@ -1,8 +1,30 @@
-import React, { useContext, useState } from 'react';
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
Seems excessive and should be only used by line unless we really should ignore it

Reply:
Changed them to by line